stockflux-securities-master/src/components/reusableComponents/ToggleSwitch.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./ToggleSwitch.css";
+
+const TextField = ({ value, id, checked, onChange }) => {
```

Comment:
Shouldn't this be `ToggleSwitch`?

Reply:
Oh dear, such a bad mistake. Yes it should...

Reply:
changed now